### ISSUE TITLE
Web Search Svelte (javascript) error - ReferenceError: responseMessageId is not defined

### ISSUE THREAD

**Issue by 238SAMIxD** [open-webui/open-webui#5672]
# Bug Report

## Installation Method

Docker
```bash
sudo docker run -d --network=host -v open-webui:/app/backend/data -e OLLAMA_BASE_URL=http://127.0.0.1:11434 --name open-webui --restart always ghcr.io/open-webui/open-webui:main
```

## Environment

- **Open WebUI Version:** v0.3.26
- **Ollama (if applicable):** 0.3.11

- **Operating System:** Ubuntu 24.02 on Windows 10 WSL
- **Browser (if applicable):** Google Chrome 128.0.6613.120 and Mozilla Firefox 130.0.1

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

It should search the web and feed the llm with prompt.

## Actual Behavior:

There is a javascript error in the console.

## Description

**Bug Summary:**
I was on version v0.3.23 on my old WSL. I created a new instance and installed everything from scratch. In the newest version Web Search is not working as expected. I had the same settings all over WebUI when it worked.

## Reproduction Details

**Steps to Reproduce:**
- Setup Web Search with Google PSE
- Click on `+` button to open a menu
- Activate Web Search
- Enter the prompt

## Logs and Screenshots

**Browser Console Logs:**
```
svelte:58
    m root.svelte:51
    m +layout.svelte:238
    p +layout.svelte:235
    dt scheduler.js:119
    _t scheduler.js:79
    promise callback*ut scheduler.js:20
    ht Component.js:81
    ctx Component.js:139
    nf +layout.svelte:202
    J utils.js:41
    _t Component.js:47
    _t scheduler.js:99
    promise callback*ut scheduler.js:20
    ht Component.js:81
    ctx Component.js:139
    Ao +layout.svelte:189
    J utils.js:41
    _t Component.js:47
    _t scheduler.js:99
    Pt Component.js:164
    he root.svelte:23
    Qe client.js:434
    Y client.js:1402
    Ze client.js:367
    Wt client.js:1724
    tn client.js:295
    <anonymous> (index):101
    promise callback* 
```
**Docker Container Logs:**
*not applicable*

**Screenshots/Screen Recordings (if applicable):**
Chrome
![image](https://github.com/user-attachments/assets/5771e0e7-5b09-4eb7-814f-9e38ef851799)
Firefox
![image](https://github.com/user-attachments/assets/a06b958e-beb5-45b0-b835-c3b6f613b9d9)
![image](https://github.com/user-attachments/assets/b5e03b13-4151-4816-a746-367a04e12b0d)

## Additional Information

It is pointing to the last line of this code
```js
const getWebSearchResults = async (model: string, parentId: string, responseId: string) => {
	const responseMessage = history.messages[responseId];
	const userMessage = history.messages[parentId];

	responseMessage.statusHistory = [
		{
			done: false,
			action: 'web_search',
			description: $i18n.t('Generating search query')
		}
	];
	history.messages[responseMessageId] = responseMessage;
	(...)
```

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by graelo** [comment]
Exact same issue I was about to raise, thanks @238SAMIxD 

The was introduced in v0.3.24: I just checked again v0.3.23 worked. It prevents websearch from working correctly, maybe other features are affected, but none of the ones I currently use.

Thanks maintainers!! ❤️

**Comment by tjbck** [comment]
Should be fixed on dev!

**Comment by 238SAMIxD** [comment]
Thank you, I closed it then. I do not need it now so I can just wait till the new release <3

**Comment by graelo** [comment]
I can confirm the issue is fixed in `v0.3.28` . Thanks @tjbck!
